docs/API/pep_actions.md

```diff
@@ -0,0 +1,36 @@
+<a name="actions"></a>
```

Comment:
It seems this file has been copied for the existing one at https://github.com/telefonicaid/perseo-fe/blob/master/documentation/pep_actions.md

No problem with it (moving .md to other directories to imprevo structure is fine) but please ensure the old one is removed. That way github will detect it as a rename and the diff will be shorter and easier to review.

Reply:
(Not only this file, probably it hapens the same with the ones in this PR)

Reply:
Yes... Fixed in 45e1084